css/styles.css

```diff
@@ -1,715 +1,1549 @@
+  justify-content: space-between;
+}
+
+.more__link {
+  font-size: 12px;
+  width: 122px;
+  font-weight: bold;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  position: relative;
+}
+
+.more__link::after {
+  content: '';
+  position: absolute;
+  top: 3px;
+  right: 0;
+  width: 30px;
+  height: 8px;
+  background: url("../images/logo/arrow-b.svg") no-repeat;
+  transition: background .5s;
+}
+
+.more__link:hover::after,
+.more__link:focus::after {
+  background: url("../images/logo/arrow-b-red.svg");
+}
+
+.more__data {
+  font-size: 12px;
+  color: #b3b3b3;
+  letter-spacing: .3px;
+}
+
+.section-stories-bottom {
+  margin: 0 auto;
+}
+
+.stories-bottom__hashtag {
+  text-align: right;
+  text-transform: uppercase;
+  font-size: 18px;
+  color: #b3b3b3;
+  font-weight: bold;
+  margin: 0 0 8px 0;
+}
+
+.pictures {
+  display: flex;
+  justify-content: center;
+}
+
+.pictures__image {
+  width: 147px;
+}
+
+.pictures__image:last-child {
+  display: none;
+}
+
+@media screen and (max-width: 1024px) {
+  .section-stories {
+    margin: 0 auto 94px;
+  }
+
+  .section-stories-top {
+    margin: 0 auto 87px;
+  }
+
+  .stories-content {
+    width: 276px;
+  }
+
+  .stories-content__title {
+    margin: 26px 0 7px 0;
+  }
+
+  .stories-content__text {
+    font-size: 13px;
+    line-height: 16px;
+    width: 277px;
+    margin: 0 0 22px 0;
+  }
+
+  .stories-content__photo {
+    width: 277px;
+    height: 149px;
+  }
+
+  .more__link {
+    width: 110px;
+  }
+
+  .pictures__image {
+    width: 133px;
+    height: 135px;
+  }
+
+  .pictures__image_display-none {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 940px) {
+  .section-stories-top {
+    justify-content: space-around;
+  }
+
+  .stories-content:last-child {
+    display: none;
+  }
+
+  .pictures__image:nth-child(6),
+  .pictures__image:nth-child(7) {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .section-stories {
+    background-color: #000;
+    color: #fff;
+    padding: 63px 0 0 0;
+    position: relative;
+    margin: 0 auto 353px;
+  }
+
+  .section-stories__title {
+    font-size: 30px;
+  }
+
+  .section-stories-top {
+    justify-content: flex-start;
+    padding: 0 0 0 43px;
+  }
+
+  .stories-content {
+    width: 298px;
+    margin: 0 71px 0 0;
+  }
+
+  .stories-content__title {
+    font-size: 25px;
+    margin: 26px 0 7px 0;
+  }
+
+  .stories-content__text {
+    font-size: 20px;
+    line-height: 25px;
+    width: 312px;
+    margin: 0 0 37px 0;
+  }
+
+  .stories-content__photo {
+    width: 312px;
+    height: 165px;
+  }
+
+  .more__link {
+    font-size: 20px;
+    width: 204px;
+    color: #fff;
+  }
+
+  .more__link::after {
+    background: url("../images/logo/arrow-w.svg") right no-repeat;
+    background-size: cover;
+    width: 49px;
+    height: 16px;
+    top: 5px;
+  }
+
+  .more__data {
+    font-size: 15px;
+    margin: 4px -33px 0 0;
+  }
+
+  .stories-content:nth-child(2) 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Itogovya_axion/pullRequest?prid=6118030)